src/Core/Content/Product/Aggregate/ProductCrossSelling/ProductCrossSellingEntity.php

```diff
@@ -32,7 +32,7 @@ class ProductCrossSellingEntity extends Entity
 
     protected ?ProductEntity $product = null;
 
-    protected string $productStreamId;
+    protected ?string $productStreamId = null;
```

Comment:
should we adjust the setter also? the getter is already right :grimacing:

Reply:
that is a pattern we have some times, you are not allowed to set it to null, as the getter was correct I didn't want to change that 🤷